lib/import-export-visitor.js

```diff
@@ -184,8 +184,8 @@ class ImportExportVisitor extends Visitor {
   visitExportAllDeclaration(path) {
     const decl = path.getValue();
     const hoistedCode =
-      pad(this, "module.importSync(", decl.start, decl.source.start) +
-      getSourceString(this, decl) +
+      pad(this, "module.watch(", decl.start, decl.source.start) +
+      "require(" + getSourceString(this, decl) + ")" +
```

Comment:
Should the `"require(" + getSourceString(this, decl) + ")"` bit be in either the first `pad()` or the following one?